src/main/java/com/google/sps/data/Suggestion.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+/** An item for a suggestion */
+public final class Suggestion {
+
+  private final String platform;
+  private final String addition;
+  private final long timestamp;
+
+  public Suggestion( String platform, String addition, long timestamp) {
+    this.platform = platform;
+    this.addition = addition;
+    this.timestamp = timestamp;
+  }
+}
```

Comment:
Style nit: Always add a line break at the end. (Add to code linter if possible.)

Reply:
Added line breaks to end of files.